srcpkgs/xorgproto/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'xorgproto'
 pkgname=xorgproto
-version=2020.1
+version=2021.4.99.2
```

Comment:
I tend to read "99" as beta releases. Is that not the case?

Reply:
Yes, I'd suggest to use 2021.4 instead.

Reply:
Done.